benchmarks/backend_request_func.py

```diff
@@ -249,14 +249,15 @@ async def async_request_openai_completions(
                 if request_func_input.model_name else request_func_input.model,
             "prompt": request_func_input.prompt,
             "temperature": 0.0,
-            "best_of": request_func_input.best_of,
             "max_tokens": request_func_input.output_len,
             "logprobs": request_func_input.logprobs,
             "stream": True,
             "stream_options": {
                 "include_usage": True,
             },
         }
+        if request_func_input.best_of > 1:
+            payload["best_of"] = request_func_input.best_of
```

Comment:
This can be reverted once this merges: https://github.com/vllm-project/vllm/pull/14299

Reply:
or https://github.com/vllm-project/vllm/pull/13997